cache/util/getRetryStrategy.js

```diff
@@ -0,0 +1,12 @@
+// simple retry strategy
+
+// default max delay in milliseconds
+const DEFAULT_MAX_DELAY = 10000
+
+module.exports = function getRetryStrategy (maxDelay) {
+  const maximumDelay = maxDelay || DEFAULT_MAX_DELAY
+
+  return function (retryCount) {
+    return Math.min(1000 + (retryCount * 1000), maximumDelay)
+  }
+}
```

Comment:
I opted to use a simple retry strategy instead of managing the reconnections manually. If we find the need to manually handle the reconnects, we can switch it out later.